1.cpp

```diff
@@ -0,0 +1,7 @@
+//Write a program to print  “Welcome to Girl Code It”?
+#include<iostream>
+using namespace std;
+int main()
+{
+    cout<<"Welcome to Girl Code It"<<endl;
+}
```

Comment:
delete these as well. @Swapna804

Reply:
Could you please check. I  am new to open source so i am not sure

Reply:
There are still 5 extra files in the root folder which are outside your name folder. Delete them too and your PR will be ready to merge 😄

Reply:
how to know where these files are